docs/docs/adrs/adr-017-allowing-inactive-validators.md

```diff
@@ -0,0 +1,111 @@
+to validate on consumer chains.
+
+In the context of this ADR, "consensus validator set" is the set of validators participating in the consensus protocol, and "staking validator set" is the set of validators viewed as active by the staking module.
+
+Currently, the staking module, provider module, and CometBFT interact in this way:
+
+![inactivevals_before.png](../../figures/inactivevals_before.png)
+
+The staking module keeps a list of bonded validators. It sends these validators to CometBFT to inform which validators make up the next consensus validators, that is, the set of validators participating in the consensus process. Separately, the provider module reads the list of bonded validators and sends this to the consumer chain, after shaping it according to which validators are opted in and the parameters set by the consumer chain for allowlist, denylist, etc.
+
+## Decision
+
+The proposed solution to allow validators that are not participating in the consensus process on the provider (inactive validators) is to:
+
+a) increase the number of bonded validators in the staking module to a large number, e.g. 500 (vs 180 today).
+This would usually mean that all those 500 validators take part in the consensus process on the provider, and we do not want this (since it slows down consensus and dilutes rewards),
+so to handle this, we also:
+
+b) do *not* take the updates for CometBFT directly from the bonded validators in the staking module, and instead *filter* the bonded validators to send only the first `MaxProviderConsensusValidators` (sorted by largest amount of stake first) many validators to CometBFT. Practically, we achieve this by repurposing the provider module, reading the bonded validators from the staking module, and simply filtering that list before sending it to CometBFT.
+
+Then lastly, we
+c) use the enlarged list of bonded validators from the staking module as basis for the validator set that the provider module sends to consumer chains (again after applying power shaping and filtering out validatiors that are not opted in).
+
+In consequence, the provider chain can keep a reasonably-sized validator set, while giving consumer chains a much larger pool of potential validators.
+
+![inactivevals_after.png](../../figures/inactivevals_after.png)
+
+
+Some additional considerations:
+
+* Migration: In the migration, the last consensus validator set will be set to the last active validator set from the view of the staking module. Existing consumer chains are migrated to have a validator set size cap (otherwise, they could end up with a huge validator set including all the staking-but-not-consensus-active validators from the provider chain)
+* Slashing: Validators that are not part of the active set on the provider chain can still be slashed for downtime on a consumer chain, but they *are not* slashed for downtime on the provider chain.
```

Comment:
What does it mean "slashed for downtime on a consumer chain"? Slashing should only take place on the provider.